Tahoe style certs for PennState

bryanlandia opened the pull request.

bryanlandia committed:
WIP Tahoe-style certs

bryanlandia committed:
Add Tahoe-style certs CSS, tweak styles for Penn State

bryanlandia committed:
Use a better certs server-error template with PSU contact info

bryanlandia commented:
Updated to rebase off of unmerged amir/ginkgo.psu/BLACK-700 since @amirtds and I are both working on changes to push to staging

amirtds commented on lms/templates/theme-variables.html:
```diff
@@ -1,17 +1,42 @@
+<%! from django.utils.functional import Promise %>
+<%! from django.utils import six %>
+<%! from django.utils.functional import lazy %>
+<%! from datetime import date %>
```
@bryanlandia I think this should be `<%! from datetime import datetime, date %>` We use `datetime.now()` to get current year https://github.com/appsembler/edx-theme-customers/blob/648708c85452717577b3e473ab57c4b9a0f4cb33/lms/templates/theme-variables.html#L166

bryanlandia committed:
Add missing datetime import to theme-variables.html

amirtds reviewed (approved):
Thanks @bryanlandia It looks good. I checked quickly the staging server here https://learn.extension-dev.psu.edu/certificates/5cfe5a84c7074358adb2b516c544b9b3 and looks good

bryanlandia commented:
I'm not merging yet since we don't have any approval yet from PSU and don't want these deployed to prod accidentally.

bryanlandia committed:
Update with new certificate text

bryanlandia committed:
Updated cert layout
no accomplishment symbol
align accomplishment title with top (don't center full cert)
assign (always same) header and text for About platform
assign (always same) header and text for About accomplishment

bryanlandia commented:
@amirtds  I'm going to merge this in advance of probable production push Wednesday or Thursday of this week.  I rebased off of ginkgo/pennstate so it's got your latest changes.
